modules/library/metadata/src/main/java/org/geotools/util/logging/Log4JLogger.java

```diff
@@ -49,37 +82,37 @@ public Log4JLogger(final String name, final org.apache.log4j.Logger logger) {
     private static org.apache.log4j.Level toLog4JLevel(final Level level) {
         final int n = level.intValue();
         switch (n / 100) {
-            default:
-                {
-                    // MAX_VALUE is a special value for Level.OFF. Otherwise and
-                    // if positive, log to fatal since we are greater than SEVERE.
-                    switch (n) {
-                        default:
-                            if (n >= 0)
-                                return org.apache.log4j.Level.FATAL; // fallthrough ALL otherwise.
-                        case Integer.MIN_VALUE:
-                            return org.apache.log4j.Level.ALL;
-                        case Integer.MAX_VALUE:
-                            return org.apache.log4j.Level.OFF;
-                    }
-                }
-            case 10:
-                return org.apache.log4j.Level.ERROR; // SEVERE
-            case 9:
-                return org.apache.log4j.Level.WARN; // WARNING
+            case 10: // SEVERE
+                return org.apache.log4j.Level.ERROR;
+            case 9: // WARNING
+                return org.apache.log4j.Level.WARN;
             case 8: // INFO
-            case 7:
-                return org.apache.log4j.Level.INFO; // CONFIG
+                return org.apache.log4j.Level.INFO;
+            case 7: // CONFIG
+                return CONFIG;
             case 6: // (not allocated)
-            case 5:
-                return org.apache.log4j.Level.DEBUG; // FINE
-            case 4:
-                return org.apache.log4j.Level.TRACE; // FINER
+            case 5: // FINE
+                return org.apache.log4j.Level.DEBUG;
+            case 4: // FINER
+                return org.apache.log4j.Level.TRACE;
             case 3: // FINEST
+                return FINEST;
             case 2: // (not allocated)
             case 1: // (not allocated)
```

Comment:
are these two supposed to fall through to 0?

Reply:
Yes, `ALL`.